code/src/main/java/com/google/sps/data/PersonEmail.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+
+public class PersonEmail implements ContactStatus{
+
+  String nameOfSender;
+  String email;
+  long getTimeWhenEmailedUnixTimeSeconds;
+  boolean personHasBeenEmailed;
+
+  public PersonEmail(String nameOfSender, String email) {
+    this.nameOfSender = nameOfSender;
+    this.email = email;
+  }
+  
```

Comment:
Feels like you should override setSuccess, getStatus, and getTime... even if they are empty in the interface.